src/components/footer/footer.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+
+import Icon from './icon';
+import facebook from './facebook.svg';
+import github from './github.svg';
+import medium from './medium.svg';
+import twitter from './twitter.svg';
+
+const styles = theme => ({
+	container: {
+		backgroundColor: '#352A3A'
```

Comment:
@jodymlin Can you help me check if we use the same color in the `Banner` component as well? This might be a hint that we should put this color code in the `theme` object. If so, please submit another commit that abstract this away in `theme` and make changes where else necessary.